tests/regression/test_sampleUseOfModel.py

```diff
@@ -13,7 +13,7 @@ def test_cli_run(base_data_dir):
         hdf5_to_csv.main([str(h5_file), str(csv_file)])
         baseline = create_baseline(csv_file)
 
-        test_df = pd.read_csv(csv_file)
+        test_df = pd.read_csv(csv_file).drop(columns="std").rename(columns={"mean": "total"})
```

Comment:
does this change to total just mean we don't need to change more things in the tests further down?  If so I'm on board.

Reply:
Yeah, I wanted to show that the values are actually the same, it was just a change in the column name. But maybe it's better to just update the file. The tests passed, so you know that my changes are not changing the values now :)

Reply:
The diff won't look like just one line diff, because there's actually a new column now, so all the values will be modified due to that.